### PR TITLE
add weightwatchers plugin

### PR DESCRIPTION
**Pull request by cdreetz**
# TL;DR
New plugin for using weightwatcher model analyzer

## Type
 - [ ] Bug Fix
 - [ ] Feature
 - [x] Plugin

## Are all requirements met?

 - [x] Code completed
 - [x] Smoke tested
 - [x] Unit tests added
 - [ ] Code documentation added
 - [ ] Any pending items have an associated Issue

## Complete description
 Add weightwatcher plugin based on get started tutorial here https://github.com/CalculatedContent/WeightWatcher
User has a model, applies 'Analyze' and is returned a df on the layer details of the model, displayed in Decks

## Tracking Issue
https://github.com/flyteorg/flyte/issues/3013

## Follow-up issue
_NA_

### CODE REVIEW

File: plugins/flytekit-deck-weightwatcher/README.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# Flytekit Deck Plugin
+
+This Plugin provides more renderers to improve task visibility.
+
+To install the plugin, run the following command:
+
+```bash
+pip install flytekitplugins-deck-standard
```
Comment (kumare3) [inline plugins/flytekit-deck-weightwatcher/README.md:8]: this should be changed

**Comment by kumare3** [comment]
@cdreetz is this ready to be reviewed? cc @eapolinario / @pingsutw did any of you review this

**Comment by kumare3** [comment]
cc @cosmicBboy
